decidim-core/app/cells/decidim/footer_topics_cell.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  # @todo Document this cell
+  #
+  # Example:
+  #
+  #    cell("decidim/footer_topics")
```

Comment:
If you change the call to what we have now it'll give an error, as cells must have always a parameter. In other cases we just pass them nil. 

```suggestion
  #    cell("decidim/footer_topics", nil)
```